vm/src/error.rs

```diff
@@ -0,0 +1,37 @@
+use rustpython_parser::ast;
+use rustpython_parser::error::ParseError;
+
+use std::error::Error;
+use std::fmt;
+
+#[derive(Debug)]
+pub enum CompileError {
+    // Cannot store value in target
+    Assign(ast::Expression),
```

Comment:
Instead of storing the expression here, insert its location only, then we do not require the clone attribute on Expression.

Reply:
Ah nice one. Will do